cmd/hz/thrift/ast.go

```diff
@@ -88,13 +97,13 @@ func astToService(ast *parser.Thrift, resolver *Resolver) ([]*generator.Service,
 
 			var reqName string
 			if len(m.Arguments) >= 1 {
+				if len(m.Arguments) > 1 {
+					return nil, fmt.Errorf("function '%s' has more than one argument, but only the first can be used in hertz now", m.GetName())
```

Comment:
If `only the first can be used in hertz now` this should only be a warning message, not a error?

Reply:
done